### PR TITLE
Card Viewer in chat

### LINKED ISSUE #142: MtG Cards from the recognizer
As you may know, I have been working on a system for automatically recognizing MtG cards during AFK streams and showing them on the overlay. I'm still evaluating the best option for the actual recognition tech, but the process for transmitting that information to the overlay is pretty much figured out. XSplit is a little tricky to get information into, so I am using a websocket system with the publisher being a php script on the local computer and a javascript subscriber as a plugin in xsplit (much in the way we are doing the twitch sub notifier). For the sake of simplicity, I'm using PubNub to provide the websocket server as they have a simple API and libraries for pretty much any language.

During the MTG AFK streams, I often see people in the chat doing the !card command for the card on the overlay. I was thinking that, if lrrbot had the PubNub subscribe key and channel name, it could just automatically report the current card in the chat (or on a separate page). Do you think that is something people would find useful?

### PR DESCRIPTION
**Pull request by mrphlip**
Closes #142

### CODE REVIEW

**Comment by mrphlip** [comment]
Just getting the pull request ready... wait to merge until we've gotten the sub key, and had a chance to test it with @paul-lrr.

File: lrrbot/commands/card.py [hunk 1]
```diff
@@ -5,12 +5,21 @@
 from lrrbot.main import bot
 
 
-with open("mtgcards.json") as fp:
-	# format:
-	# CARD_DATA = [
-	#   ('searchable name', 'display name', 'display text')
-	# ]
-	CARD_DATA = json.load(fp)
+cardview = False
```
Comment (andreasots) [inline lrrbot/commands/card.py]: I think this should be in `lrrbot.main.LRRBot`.

File: sql/cards.tbl [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    filteredname VARCHAR(255) NOT NULL,
+    name TEXT COLLATE "en_US.utf8" NOT NULL,
+    text TEXT COLLATE "en_US.utf8" NOT NULL,
+    lastprinted DATE
+);
+CREATE UNIQUE INDEX cards_name_idx ON cards(filteredname);
+
+CREATE TABLE IF NOT EXISTS card_multiverse (
+    multiverseid INTEGER NOT NULL PRIMARY KEY,
+    cardid INTEGER NOT NULL
```
Comment (andreasots) [inline sql/cards.tbl]: This is a foreign key so `cardid INTEGER NOT NULL REFERENCES(cards) ON DELETE CASCADE` and maybe `ON UPDATE CASCADE` as well?

**Comment by andreasots** [comment]
Rebased, merging.

**Pull request merged**